specs/avatar.md

```diff
@@ -0,0 +1,105 @@
+# Avatar
+
+## Overview
+
+An avatar is a graphical represention of a user or object.
+
+### Use Cases
+
+A common use case would be to display an image or text (usually initials) of a user or an object.
+
+### Features
+- Image slot: Able to slot in an avatar image
+- Badge slot: Able to slot in a badge component
+- `size`, a numeric value that determines the size of the avatar
+- `shape`, a circle or square shape can be chosen
```

Comment:
In the case of `square` would the corners be rounded based on the design system properties?

Reply:
Yes, that is my approach since radius can vary.

Reply:
I added a sentence to clarify this.